build.sh

```diff
@@ -184,6 +192,7 @@ openslidejava_upurl="https://github.com/openslide/openslide-java/tags"
 
 # Update-checking regexes
 zlib_upregex="source code, version ([0-9.]+)"
+libzip_upregex="Current Version: ([0-9].[0-9].[0-9])"
```

Comment:
This fails because of the capital V.  (Test with `./build.sh updates`.  If no libzip line is printed, you're correctly fetching a version number.)  It also won't match a version number like 1.0.10 because of the missing `+` specifiers.  I'd suggest matching on the tarball filename instead, since it's more robust to minor changes on the website.  See the other upregexes for examples.